src/pages/15Years/15Years.vue

```diff
@@ -50,6 +54,20 @@ export default {
 	metaInfo: {
 		title: '15 Years'
 	},
+	props: {
```

Comment:
Since this is a page component, I think these would make more sense as data fields and be set in this component instead of the story. Thoughts on that, anyone?

Reply:
I'll make that change